dracut.sh

```diff
@@ -272,6 +272,10 @@ Creates initial ramdisk images for preloading modules
                         Use [FILE] as a splash image when creating an UEFI
                          executable. Requires bitmap (.bmp) image format.
   --kernel-image [FILE] Location of the kernel image.
+  --sbat [FILE]         Location of the SBAT string to be added to .sbat.
```

Comment:
Why do you we need to save the SBAT string to a file? this option should be just `--sbat [STRING]` or `[TEXT]`, what you like most.

Reply:
Usually the sbat string is relatively long (see SBAT_DEFAULT), and not ideal to write by hand or paste in console. I think a file is better, especially if it is extracted from some other PE already.

Reply:
Hmm... I'd prefer not to use a file for this, but let's see what others think.

Reply:
ok

Reply:
FWIW, if your main concern handling the string as a command line option, it could also be passed using the `sbat=` variable in a conf file, so both needs would be covered.

Reply:
@aafeijoo-suse  please check if now is better